server/models/db/worker.js

```diff
@@ -11,6 +11,21 @@ async function getWorker(email, password) {
     return userModel.getUser(email, password, Tables.worker, Worker);
 }
 
+// This method inserts a worker account given specific information.
+async function insertWorkerAccount(firstName, lastName, type, workerID, email, password, phone, specialization) {
+    //Data recieved from the front end sign up form
+    //Insert one row into a table
+    return db.run(`insert into ${Tables.worker}(firstName, lastName, type, workerID, email, password, phone, specialization) values 
```

Comment:
Hi Nancy,
Please refer to the `005.do.create-worker-table.sql` for the proper naming of the table columns. For example, the attribute in the sql table is `worker_id` and not `workerID` so this may not be inserting correctly. I would suggest adjusting the naming to ensure that it inserts properly.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've changed the following discrepancies in naming of the table columns: first_name, last_name,  worker_id. These are now in sync with the names in `005.do.create-worker-table.sql`.

You can now see these changes the following locations:

line 15 of server/models/db>worker.js
line 18 of server/models/db>worker.js
line 19 of server/models/db>worker.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, 
That's wonderful. Thank you so much for making those changes!